notify/discord/discord.go

```diff
@@ -107,6 +110,17 @@ func (n *Notifier) Notify(ctx context.Context, as ...*types.Alert) (bool, error)
 		color = colorGreen
 	}
 
+	var url string
+	if n.conf.WebhookURL != nil {
+		url = n.conf.WebhookURL.String()
+	} else {
+		content, err := os.ReadFile(n.conf.WebhookURLFile)
```

Comment:
A bit of a nit, but perhaps call this `b` instead of `content`. Two reasons:

1. It's short.
2. It's a somewhat idiomatic way of telling other maintainers that this variable is a byte slice instead of a string without having to check the docs for `os.ReadFile`.

```suggestion
		b, err := os.ReadFile(n.conf.WebhookURLFile)
```